.travis.yml

```diff
@@ -95,8 +95,10 @@ jobs:
         - export CONDA_PATH="\$env:Path = ';C:\miniconda3;C:\miniconda3\Library\mingw-w64\bin;C:\miniconda3\Library\usr\bin;C:\miniconda3\Library\bin;C:\miniconda3\Scripts;C:\miniconda3\bin;C:\miniconda3\condabin;C:\iverilog\bin;' + \$env:Path"
         - powershell "$CONDA_PATH; conda install --yes -c msys2 m2-base m2-make m2w64-toolchain libpython"
         - powershell "$CONDA_PATH; conda install --yes virtualenv"
+        - powershell "$CONDA_PATH; conda update --yes pip"
+        - powershell "$CONDA_PATH; conda info; conda list --show-channel-urls"
         # needed to make the builtin sqlite3 module (used by coverage) work - see gh-1909
-        - powershell '$CONDA_PATH; cp C:\miniconda3\Library\bin\sqlite3.dll C:\miniconda3\DLLs\'
+        - powershell 'cp C:\miniconda3\Library\bin\sqlite3.dll C:\miniconda3\DLLs\sqlite3.dll'
```

Comment:
Was this change necessary?

Reply:
No, it's a cleanup (and in a separate commit). $CONDA_PATH was echoed as-is without variable substitution, so it wasn't doing any harm, but also not doing anything useful.

Reply:
Well, it might be pointing `cp` to something else, but I suppose that's awfully contrived.

Hadn't noticed the separate commit. Changing the `\` to `/` and using double quotes would be more consistent, but not a big deal.